airbyte-integrations/connectors/source-s3/integration_tests/spec.json

```diff
@@ -365,6 +365,13 @@
         "order": 3,
         "type": "string"
       },
+      "role_arn": {
+        "title": "AWS Role ARN (External ID is '8eee93517501b43a')",
+        "description": "Specifies the Amazon Resource Name (ARN) of an IAM role that you want to use to perform operations requested using this profile. Set External ID as '8eee93517501b43a'.",
+        "airbyte_secret": true,
```

Comment:
Spoke about this privately, but I am in favor of not setting this to secret. This value is not sensitive and I think it is helpful to be able to see it. Very open to push back here though.

Reply:
I agree with you, as role arn cannot be used outside the users' Airbyte workspace, it should not be secret. I fixed it in the latest commit.